protocolport/protocol_port.go

```diff
@@ -0,0 +1,7 @@
+package protocolport
+
+type ProtocolPort struct {
+	IngressPort string `json:"ingressPort" yaml:"ingressPort"`
+	LBPort      string `json:"lbPort" yaml:"lbPort"`
+	Protocol    string `json:"protocol" yaml:"protocol"`
```

Comment:
I deal with other problems at the moment so this is not my priority. I need my focus at the moment.